app/query.go

```diff
@@ -129,12 +129,12 @@ func QueryReceipts(addr string, height int64) (proofs []pocketTypes.Receipt, err
 	return pocket.QueryReceipts(Codec(), getTMClient(), a, height)
 }
 
-func QueryReceipt(blockchain, appPubKey, addr string, sessionblockHeight, height int64) (proof *pocketTypes.Receipt, err error) {
+func QueryReceipt(blockchain, appPubKey, addr, receipt string, sessionblockHeight, height int64) (proof *pocketTypes.Receipt, err error) {
```

Comment:
Why querying a receipt (`pocketTypes.Receipt`) requires a `receipt string`

Reply:
Ah that's actually receiptType -> `relay` or `challenge` will edit